pkg/types/alibabacloud/platform.go

```diff
@@ -7,11 +7,32 @@ type Platform struct {
 
 	// ResourceGroupID is the ID of an already existing resource group where the cluster should be installed.
 	// This resource group must be empty with no other resources when trying to use it for creating a cluster.
```

Comment:
Is this strictly true? Or could the user-supplied VPC be in the resource group?

Reply:
In fact, it has no effect if it is not empty. we will not query and destroy resources through resource group. So this description is incorrect, I will delete it.

Reply:
Will the destroyer delete the resource group if it is user-supplied?